docs/Analog.md

```diff
@@ -1,32 +1,90 @@
 
-## PCB Layout
+| Pin number | Signal name |
+|------------|--------|
+| 1 | `Analog P1` |
+| 2 | `Analog N1` |
+| 3 | +15V |
+| 4 | `GND` |
+| 5 | -15V |
+| 6 | `GND` |
+| 7 | `Analog P2` |
+| 8 | `Analog N2` |
+
+The location of these 4 analog connectors in the AMDC (stacked as 2 x 2) is shown below:
+
+<img src="images/amdc-analog-input-highlighted.svg" />
+
+In this picture, the top left connector (when looking from the right side) corresponds to the `Analog connector 1` in the block diagram, the top right connector is the `Analog connector 2`, the bottom left connector is the `Analog connector 3`, and the bottom right connector is the `Analog connector 4`.
+
+More detailed information on the analog connectors can be found in the [datasheet](https://media.digikey.com/pdf/Data%20Sheets/Amphenol%20PDFs/RJSAE_Brochure.pdf).
+
+### Difference amplifiers
+Analog input signals should be conditioned before transmitting to the ADC. The first stage of this conditioning is to decrease the voltage level. These are implemented using 4 [INA2143UA](http://www.ti.com/lit/ds/symlink/ina143.pdf) difference amplifier chips. Each chip has two op amps, totaling in 8 op amps each of which receives one differential analog input. These chips need a ±15V supply which they receive from the AMDC. Each op amp is configured as a non-inverting amplifier and the relationship between its input and output is as follows:
+
+
+_V_<sub>i</sub> = _V_<sub>REF</sub>  + 0.1 (_V_<sub>i</sub><sup>+</sup> - _V_<sub>i</sub><sup>-</sup>)
 
-Discussion related to PCB layout for analog subsystem:
-- What area of board is it on?
-- Noise??
+where i = 1, 2, 3 ..., 8. (_V_<sub>i</sub><sup>+</sup> - _V_<sub>i</sub><sup>-</sup>) is the bipolar differential input coming from analog connectors and _V_<sub>i</sub> is the output of the op amp which is fed to the low-pass filter (next subsection). _V_<sub>REF</sub> = 2.048V is supplied from the AMDC and 0.1 gain is set using the laser trimmed resistors during IC fab, so each resistance value internal to the [INA2143UA](http://www.ti.com/lit/ds/symlink/ina143.pdf) is very accurate. This results in a very high CMRR. The differential input to the op amp can be between ±20V (AC/DC, each input between ±10V), which means the output can be in the range of 0.048V and 4.048V. The voltage level at each stage of the analog signal chain is shown by a red color in the block diagram.
 
-## Max Voltage Limitations
+More detailed information on the operating conditions of the op amp can be found in the [datasheet](http://www.ti.com/lit/ds/symlink/ina143.pdf).
+ 
+### Low-Pass Filters (LPFs)
+After the analog input voltage levels are decreased using the op amps, the next stage is to remove the high-frequency noise using the low-pass filters. To implement this, 8 simple first-order RC filters are used each for 1 analog input. The cutoff frequency of the filter is selected based on the resistance and capacitance values:

```

Comment:
Please provide a diagram here that explains the low-pass filter stage. I suggest something similar to this:

![image](https://user-images.githubusercontent.com/7421348/82832075-bf8dd500-9e7f-11ea-881a-d236a99754ea.png)